ldap/servers/slapd/back-ldbm/findentry.c

```diff
@@ -292,9 +306,14 @@ find_entry_internal_uniqueid(
                       uniqueid);
     }
 
-    /* entry not found */
-    slapi_send_ldap_result(pb, (0 == err || DBI_RC_NOTFOUND == err) ? LDAP_NO_SUCH_OBJECT : LDAP_OPERATIONS_ERROR, NULL /* matched */, NULL,
-                           0, NULL);
+    if (reverted_entry) {
+        slapi_send_ldap_result(pb, LDAP_BUSY, NULL, NULL, 0, NULL);
```

Comment:
IMHO We should add a rexr message to explain why LDAP_BUSY was returned. 
 (i.e:  slapi_send_ldap_result(pb, LDAP_BUSY, "target entry busy because of a cancelled operation.", NULL, 0, NULL);

Reply:
yes good point